### PR TITLE
LPS-91162 | master

### PR DESCRIPTION
**Pull request by patriciajeanperez**

### CODE REVIEW

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/cdbdea591a...1ea2809708

**Comment by brianchandotcom** [comment]
@patriciajeanperez @boton can you rephrase 8998702d814e9fac5ed4262ddfaddbacb0f33883 if my change is wrong? I just wanted to make it a full sentence.

**Comment by boton** [comment]
@brianchandotcom to @drakonux and me sound good
